### ISSUE TITLE
Configure babel-loader to include both the app source path and node_modules/@material

### ISSUE THREAD

**Issue by jperera84** [Polymer/pwa-starter-kit#160]
I'm trying to integrate mdc components in an application, so far so good with several other mdc components but I found an issue with the `@material\dialog`.
I have this comment from mdc team:

> When I install using your list of dependencies, I also get focus-trap@2.4.6 under top-level node_modules.
> 
> I'm wondering if this is not so much a npm issue as a toolchain issue... how are you transpiling/serving your JS? The fact that you're getting this error in the browser console in particular has me wondering, unless that's being forwarded from a dev server.
> 
> We're using MDC Dialog in our MDC Web Catalog repository, currently locked to focus-trap@2.4.5, with no issues. AFAICT the index.js in focus-trap hasn't changed in 2.4.6. One thing worth noting is that we specifically configure babel-loader to include both the app source path and node_modules/@material.

I'm going to reference a Github issue that I created for the MDC component:
[github issue](https://github.com/material-components/material-components-web/issues/2901#issuecomment-398211813)

Please could someone take a look at this, I need to start using this component in my application but it seems to be a problem with PWA Starter kit in the way is loading the packages because I tested directly in a vanilla app and the component works fine.

Thanks

**Comment by jperera84** [comment]
Thanks, I though that the mdc components were going to be the new components for polymer is what I hear from the last Google IO conferences. So if I want to start using the material io components is not going to be possible? I have to wait until the polymer team finish the components, it doesn't seems to be a good idea.
Thanks anyway

**Comment by keanulee** [comment]
The NPM warnings shouldn't make a difference (even if you fix the peer dependencies). The other components you mentioned probably don't use the `focus-trap` library and might _compile_, but are you sure they're _styled_ properly when viewed in a browser? (I don't see how its CSS styles can apply inside shadow DOM.)

**Comment by jperera84** [comment]
Yes it works perfectly fine, what Im doing is the same that the polymer team that it is working on the **material-components-web-components** are doing. I install the component that I want to use and then I create an style component:
```
import { html } from '@polymer/lit-element';
export const FormFieldStyles = html`
<style>
.mdc-form-field{font-family:Roboto,sans-serif;-moz-osx-font-smoothing:grayscale;-webkit-font-smoothing:antialiased;font-size:.875rem;line-height:1.25rem;font-weight:400;letter-spacing:.01786em;...
</style>
```
Then I'm importing those styles webcomponents into my ```shared-styles.js``` and all my components are styled perfectly fine... This approach was the same that the polymer team is using for what I see in the code of those components, and the good thing is that I can use the component javascript logic too using:
```import { MDCTextField, MDCTextFieldFoundation } from '@material/textfield';
      async _firstRendered() {
        await this.renderComplete;
        this.tbUsuarioEmail = new MDCTextField(this.shadowRoot.querySelector("#tbUsuarioEmail"));
       }
```
And that's works perfectly... The only problem was with dialog, but I solved it using the styles but I can't use the javascript logic because of the error.
![image](https://user-images.githubusercontent.com/13193138/41751443-48d3fcc0-758f-11e8-815b-2d4865ae9ecd.png)
